.nvmrc

```diff
@@ -1 +1 @@
-14.18.0
+14
```

Comment:
needed because this is [used to derive the docker image version](https://github.com/hmrc/x-govuk-component-renderer/blob/main/docker/Dockerfile#L3) and the debian sources have changed since the base image we get when pinned to a minor version was created so docker builds using that get a load of 404s when running apt-get commands